src/client/js/Panels/ControllerBase/DiagramDesignerWidgetMultiTabMemberListControllerBase.js

```diff
@@ -1808,6 +1868,7 @@ define(['js/logger',
         this._GMEID2ComponentID[gmeID].push(uiComponent.id);
         this._ComponentID2GMEID[uiComponent.id] = gmeID;
 
+        this._delayedConnectionsAsItems = this._delayedConnectionsAsItems || {};
```

Comment:
you are right, it should be in the constructor (or actually in the initialization of the tab...)